source/about/faq-license.rst

```diff
@@ -0,0 +1,318 @@
+(c) Mattermost, Inc. 2015-present.
+
+With regard to the Mattermost Software:
+
+This software and associated documentation files (the "Software") may only be
+used in production, if you (and any entity that you represent) have agreed to,
+and are in compliance with all of the following: (a) the Mattermost Terms of Use, 
+available at https://mattermost.com/terms-of-use/ (the "TOU"), (b) and the Mattermost 
+Software License Agreement,  available at https://mattermost.com/enterprise-edition-terms/ 
+(the “SLA”) or other licensing agreement governing your use of the Software, as agreed 
+by you and Mattermost, and otherwise have a valid Mattermost Enterprise for the correct 
+number of Registered Authorized Users the Software. Subject to the foregoing, you are free
+to modify this Software and publish patches to the Software. You agree that
+Mattermost and/or its licensors (as applicable) retain all right, title and
+interest in and to all such modifications and/or patches, and all such
+modifications and/or patches may only be used, copied, modified, displayed,
+distributed, or otherwise exploited with a valid license or Subscription for the correct number of 
+Registered Authorized Users of the Software.  Notwithstanding
+the foregoing, you may copy and modify the Software for development and testing
+purposes, without requiring a valid license or Subscription.  You agree that Mattermost and/or
+its licensors (as applicable) retain all right, title and interest in and to
+all such modifications.  You are not granted any other rights beyond what is
+expressly stated herein.  Subject to the foregoing, it is forbidden to copy,
+merge, publish, distribute, sublicense, and/or sell the Software.
+
+The full text of this Source Available License shall be included in all copies or substantial
+portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+For all third party components incorporated into the Mattermost Software, those
+components are licensed under the original license provided by the owner of the
+applicable component."
+
+Open source derivative work of Mattermost
+-----------------------------------------
+
+How do I fork Mattermost?
+-------------------------
+
+If you wish to create a forked version of the Mattermost source code, you must comply with the applicable licenses under which the source code is made available. Historically, this was made clear via this link: https://docs.mattermost.com/about/faq-business.html#copyright-and-licensing-of-mattermost-server-open-source-code.
```

Comment:
Just confirming the links in this FAQ work once the PR is merged?
 - https://docs.mattermost.com/about/faq-business.html#copyright-and-licensing-of-mattermost-server-open-source-code
 - https://docs.mattermost.com/about/faq-enterprise.html#what-are-mattermost-s-policies-around-licensing-terms-of-use-and-privacy

Not sure if they'll get redirected to the right section automatically

Reply:
Thanks @justinegeffen! Just one question from my side

Reply:
I'll set those redirects up, thanks for the reminder.